package.json

```diff
@@ -22,10 +21,12 @@
     "format:yaml": "prettier --write --parser yaml '**/*.{yaml, yml}'",
     "format:graphql": "prettier --write --parser graphql '**/*.graphql'",
     "start": "yarn playground:start",
-    "prerelease": "yarn auth && yarn build && yarn typecheck",
+    "changeset": "changeset",
+    "version-packages": "changeset version",
+    "prerelease": "yarn auth && yarn build",
     "prerelease:alpha": "yarn prerelease",
     "prerelease:canary": "yarn prerelease",
-    "release": "lerna publish --exact --dist-tag next",
+    "release": "lerna exec --scope '@commercetools-{frontend,backend}/*' -- npm publish --dist-tag next",
```

Comment:
This is due to a current "limitation" of changesets.

More info here: https://github.com/atlassian/changesets/issues/356